src/alembic/versions/e74292feb010_add_profile_picture_url_to_site_user.py

```diff
@@ -0,0 +1,34 @@
+"""add profile_picture_url to site_user
+
+Revision ID: e74292feb010
+Revises: 2a6ea95342dc
+Create Date: 2024-10-04 17:23:06.869439
+
+"""
+from collections.abc import Sequence
+from typing import Union
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "e74292feb010"
+down_revision: str | None = "2a6ea95342dc"
+branch_labels: str | Sequence[str] | None = None
+depends_on: str | Sequence[str] | None = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint("fk_officer_term_computing_id_site_user", "officer_term", type_="foreignkey")
+    op.create_foreign_key(op.f("fk_officer_term_computing_id_user_session"), "officer_term", "user_session", ["computing_id"], ["computing_id"])
+    op.add_column("site_user", sa.Column("profile_picture_url", sa.Text(), nullable=True))
+    # ### end Alembic commands ###
+
+
+def downgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column("site_user", "profile_picture_url")
+    op.drop_constraint(op.f("fk_officer_term_computing_id_user_session"), "officer_term", type_="foreignkey")
+    op.create_foreign_key("fk_officer_term_computing_id_site_user", "officer_term", "site_user", ["computing_id"], ["computing_id"])
+    # ### end Alembic commands ###
```

Comment:
Please double check these constraints are consistent with the model & [naming conventions](https://github.com/CSSS/csss-site-backend/blob/77c93628230027721713911f80ff40c98b2d12c2/src/database.py#L16). The biggest thing that is not autogenerated is some kinds of constraints & triggers & stuff.

Once you've done so, make sure to remove the autogenerated comments.

Reply:
this is outdated; did u fix this?

Reply:
I didn't, you still gotta remove the comments! But feel free to merge after you do so \:)

Reply:
done! https://github.com/CSSS/csss-site-backend/pull/89/commits/ab079623efaed7658807df32746449fb247cf38a